src/views/workflow-history/workflow-history-event-status-badge/workflow-history-event-status-badge.styles.ts

```diff
@@ -0,0 +1,47 @@
+import { styled as createStyled } from 'baseui';
+
+import type { StyletronCSSObjectValue } from '@/hooks/use-styletron-classes';
+
+import getBadgeContainerSize from './helpers/get-badge-container-size';
+import { WORKFLOW_EVENT_STATUS } from './workflow-history-event-status-badge.constants';
+import type {
+  Props,
+  WorkflowEventStatusBadgeSize,
+} from './workflow-history-event-status-badge.types';
+
+export const styled = {
+  BadgeContainer: createStyled<
+    'div',
+    { $status: Props['status']; $size: WorkflowEventStatusBadgeSize }
+  >('div', ({ $size, $status, $theme }) => {
+    const containerSize = getBadgeContainerSize($theme, $size);
+
+    return {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      borderRadius: '16px',
+      width: containerSize,
+      height: containerSize,
+      ...(
+        {
+          [WORKFLOW_EVENT_STATUS.COMPLETED]: {
+            color: '#009A51',
+            backgroundColor: '#D3EFDA',
+          },
+          [WORKFLOW_EVENT_STATUS.ONGOING]: {
+            backgroundColor: '#DEE9FE',
+          },
+          [WORKFLOW_EVENT_STATUS.FAILED]: {
+            color: $theme.colors.negative,
+            backgroundColor: '#FFE1DE',
+          },
+          [WORKFLOW_EVENT_STATUS.WAITING]: {
+            color: $theme.colors.black,
+            backgroundColor: $theme.colors.backgroundSecondary,
```

Comment:
minor nit: backgroundSecondary is slightly different from the background gray we use in cadence-web

Reply:
seem like the docs is not reflecting the correct value

Reply:
Actually i found that the designs has `backgroundSecondary` as `#F3F3F3` everywhere.
While our implementation has them as `#F6F6F6` updated the value in the theme